Lengnick-Style Model/firm-structure-tests.nlogo

```diff
@@ -1,7 +1,7 @@
-extensions [table rnd]
+extensions [table rnd gis]
```

Comment:
why are you using the gis extension?

Reply:
Remnant from when import JSON file wasn't working. Was hoping it would fix the issue.